docs/README-gcp.md

```diff
@@ -50,14 +50,16 @@ To encrypt secrets using the KMS crypto key created above, follow the instructio
 ```echo -n <secret> | gcloud kms encrypt --location <location> --keyring <keyring_name> --key <key_name> --plaintext-file - --ciphertext-file - | base64```
 
 #### Encryption Using Python Script
-Alternatively, run the kms_secrets_encryption.py Python 3 script under the tools directory to automate the KMS encryption process. 
+Alternatively, the kms_secrets_encryption.py Python 3 script under the tools directory can be used to automate the KMS encryption process. 
 
-Execute the following command inside the tools directory:
+First, open the terraform.tfvars file and enter all the secrets in plaintext located under the line "# <-- Start of secrets section, do not edit this line. -->", then save the file. 
```

Comment:
can you use number list?
1. step one
2. step two, etc.

IMO it's easier to read. then you don't have to say "First,"